ui/apps/platform/src/Containers/Clusters/DelegateScanning/DelegateScanningPage.tsx

```diff
@@ -38,6 +38,26 @@ const initialDelegatedState: DelegatedRegistryConfig = {
     registries: [],
 };
 
+function addUuidstoRegistries(config: DelegatedRegistryConfig) {
+    const newRegistries = config.registries.map((registry) => {
+        // eslint-disable-next-line no-restricted-globals
+        const uuid = self.crypto.randomUUID();
```

Comment:
For my understanding, how does random UUID differ from index from `map` method here?

That is, lint for React rightly reports use of array index key during rendering, but this is on data fetch.

Reply:
I tried creating IDs from the data, but because the path and/or clusterId for each row are editable, and there is no stable ID from the API, every hack I tried had shortcomings. 
Moreover, the PatternFly drag-and-drop handlers rely on unique HTML IDs for there work, so I had `row-${index}` at one point in my iterations, with the necessary lint exception.
Linda suggested adding artificial IDs so untangle the knot of drag-and-drop not working predictably, and that serves for both the key and the ID in the JSX table markup itself.